src/js/server/handlebars.js

```diff
@@ -23,16 +26,33 @@ gpii.express.addHelper = function (that, component) {
 
 gpii.express.configureExpress = function (that, express) {
     if (that.options.config.express.views) {
-        var viewRoot = that.options.config.express.views;
+        var viewDirs     = fluid.makeArray(that.options.config.express.views);
+        var layoutDir    = null;
+        var partialsDirs = [];
+        fluid.each(viewDirs, function (viewDir) {
+            // We can only use the first layouts directory until this issue is resolved in express-handlebars:
+            //
+            // https://github.com/ericf/express-handlebars/issues/112
+            if (!layoutDir) {
+                var layoutPath = path.resolve(viewDir, "layouts");
```

Comment:
At this site you can use the little utility function you will create in dispatcher.js